Services/CO.CDP.OrganisationInformation.Persistence/Forms/FormAnswer.cs

```diff
@@ -55,11 +58,7 @@ public class FormAnswerSet : IEntityDate
 {
     public int Id { get; set; }
     public required Guid Guid { get; set; }
-
-    [ForeignKey(nameof(Organisation))]
-    public required int OrganisationId { get; set; }
-    public required Organisation Organisation { get; set; }
-
+    public required SharedConsent SharedConsent { get; set; }
```

Comment:
Do we need it here? Just a thought. Assuming we'll be always querying for FormAnswers and AnswerSets by SharedConsent, defining the relationship on one side might be sufficient?

Reply:
I think its needed